LiskHQ/lisk-hub
# Floating point must match selected language/locale - Closes #2353

### What issue have I solved?

#2353

### How have I implemented/fixed it?

- [x] Converter component;
- Amount input validations;
  - [x] Request;
  - [x] Send;
- [x] Send Form -> Fee values;
- [x] Send Summary -> values;
- [x] Delegate information (vote weight, productivity);

### How has this been tested?

It's needed to set a `feature-flag-language` into localStorage to see the option in settings. 
```js
localStorage.setItem('feature-flag-language', true);
```

- Go to Settings
- Change language
- All the values should be updated  to reflect the selected language, and also the validation of input fields.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
